### PR TITLE
Consistency in title color for blog page

### PR DESCRIPTION
**Pull request by rimshach**
I was going through the website and I found the title color different for blog's page as compared to the rest of the website.I think there should be consistency in the entire website so I changed the color

The color for titles or link in other pages:

image

The color for blog was :
image

I have changed it to :
image

### CODE REVIEW

File: antora-ui-camel/src/css/doc.css [hunk 1]
```diff
@@ -116,7 +116,7 @@
 }
 
 .doc a {
-  color: var(--link-font-color);
+  color: #303284;
```
Comment (zregvart) [inline antora-ui-camel/src/css/doc.css]: We use CSS variables, it's very difficult to maintain individual values scattered across CSS files.

Comment (rimshach) [reply antora-ui-camel/src/css/doc.css]: oh okay, there is an option under each heading link of "continue reading " ,that link and heading link redirects us to the same page, so would it be okay to remove the link from the heading ? 
Or as in the home page the link "What's New?" has been assigned color of a regular heading 
image

**Comment by rimshach** [comment]
oh alright.I actually looked at the heading link of "what's new " at home page that was consistent with rest of the headings so I thought overall headings can have same color.
For this page  there is an option under each heading link of "continue reading " ,that link and heading link redirects us to the same page, so would it be okay to remove the link from the heading ?

File: antora-ui-camel/src/css/doc.css (hunk 1 above)
Comment (rimshach) [reply antora-ui-camel/src/css/doc.css]: Oh , I'll consider this before making any further changes

**Comment by zregvart** [comment]
@rimshach I think it makes sense to remove the heading link from the blog post list as we already have the "continue reading" link there.

**Pull request closed without merging**